mne/io/tests/test_meas_info.py

```diff
@@ -541,10 +541,10 @@ def test_check_compensation_consistency():
         assert ret == expected_result
         assert len(missing) == 17
         if comp != 0:
-            with pytest.raises(RuntimeError,
-                               match='Compensation grade 1 has been applied'):
+            with catch_logging() as log:
                 Epochs(raw, events, None, -0.2, 0.2, preload=False,
                        picks=picks)
+            assert('Compensation grade 1' in log.getvalue())
```

Comment:
Even with `verbose=True` in the `Epochs` constructor (to get `info`-level messages) I get:
```
AssertionError: assert 'Compensation grade 1' in '10 matching events found\nApplying baseline correction (mode: mean)\nNot setting metadata\nRemoving 5 compensators from info because not all compensation channels were picked.\n0 projection items activated\n'
```